src/effects/effectslot.cpp

```diff
@@ -313,18 +313,6 @@ void EffectSlot::loadEffect(const EffectManifestPointer pManifest,
 
     m_pControlLoaded->forceSet(1.0);
 
-    unsigned int iNumKnobParameters = numKnobParameters();
```

Comment:
I was wondering about removing this code. Thanks for going ahead and doing it. The number of parameter slots should be fixed, otherwise skins and controller mappings don't know how many they may possibly need to support.